DOL.WHD.Section14c.Domain/Models/Submission/Attachment.cs

```diff
@@ -38,5 +38,8 @@ public Attachment()
         [IgnoreDataMember]
         public string EncryptionKey { get; set; }
 
+        public string AttachmentBlobId { get; set; }
+
+        public virtual AttachmentBlob AttachmentBlob { get; set; }
```

Comment:
It looks like this `AttachmentBlob` is not actually used anywhere?  What's it for, and why is it here if it's not being used?

Reply:
The AttachmentBlob is requested by Madhu. This table is to be used in future implementation to replace file attachments with table blobs.
- API: As an employer I would like to upload *all* current SCA wage determinations #496